### DISCUSSION TITLE
Add markdown language.

### DISCUSSION THREAD

**Discussion by YukiGasai** [monkeytypegame/monkeytype#4354] [Ideas]
Hi,
I love to train my typing on monkeytype and think especially the code languages are useful to get some real world use cases. 
It would be nice if we could get a markdown code language to train the most common features that markdown introduces, e.g. links images code blocks bold italic words.

**Comment by Miodec** [comment]
I meean, im not sure if there is much to train here, its just #, [] and () no?
